lib/schema/utils.js

```diff
@@ -389,9 +403,11 @@ const combineErrors = ({ errorCache, warningsCache, payloadErrors, stats, versio
               count: details.occurrences,
               ...(file && { fileName: file }),
               ...(pathName && { pathName }),
-              ...(lookupValue && { lookupValue }),
+              ...(details?.failedEnum != null && { lookupValue }),
```

Comment:
Are you meaning to use loose equality here? And is there a way around it?

Reply:
Was meant for checking for either undefined or null but I've made it more explicit now.